tests/unit/amo/components/TestScreenShots.js

```diff
@@ -49,7 +54,7 @@ describe('<ScreenShots />', () => {
   });
 
   it('renders custom thumbnail', () => {
-    const item = { src: 'https://foo.com/img.png', title: 'test title' };
+    const item = { src: 'https://foo.com/img.png', title: 'test title', h: HEIGHT, w: WIDTH };
```

Comment:
It would be better to pass different height/width here to make sure we use the values from `image_size`.